lms/static/sass/views/_verification.scss

```diff
@@ -165,7 +165,7 @@
 
   // elements - controls
   .action-primary {
-    @extend %btn-primary-blue;
+    @extend %btn-verify-primary;
```

Comment:
@griffresch This change also affects the button on the receipt page. Is this desired?
![receipt](https://cloud.githubusercontent.com/assets/910510/8258860/6359d8d2-1686-11e5-87c1-738cf8b27d55.png)